tests/pyfunc/test_spark.py

```diff
@@ -723,3 +726,39 @@ def test_spark_udf_stdin_scoring_server(spark, monkeypatch):
         df = spark.createDataFrame(X)
         result = df.select(udf(*X.columns)).toPandas()
         np.testing.assert_almost_equal(result.to_numpy().squeeze(), model.predict(X))
+
+
+# @pytest.mark.skipif(
+#    Version(pyspark.__version__) < Version("3.4.0"), reason="requires spark >= 3.4.0"
+# )
```

Comment:
Just for testing

Reply:
Confirmed: https://github.com/mlflow/mlflow/actions/runs/4538225945/jobs/7996927410